### PR TITLE
[PGEC-43] Rule R12

### PR DESCRIPTION
**Pull request by aomegax**
#### Motivation and Context
Rule R12:
- a CI cannot accept or request a bundle subscription if its validityDateTo is before or equal to now.

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/pagopa/afm/marketplacebe/util/CommonUtil.java [hunk 1]
```diff
@@ -29,6 +29,6 @@ public <T> PageInfo buildPageInfo(Page<T> page) {
      */
     public boolean isValidityDateToAcceptable(LocalDate validityDateTo) {
         LocalDate now = LocalDate.now();
-        return validityDateTo == null || !(validityDateTo.isEqual(now) && validityDateTo.isBefore(now));
+        return validityDateTo == null || !(validityDateTo.isEqual(now) || validityDateTo.isBefore(now));
```
Comment (jacopocarlini) [inline src/main/java/it/pagopa/afm/marketplacebe/util/CommonUtil.java:32]: this condition is always `true` except when `validityDateTo` equals to `now`

**Review by jacopocarlini — DISMISSED** [review]

File: src/main/java/it/pagopa/afm/marketplacebe/util/CommonUtil.java (hunk 1 above)
Comment (aomegax) [reply src/main/java/it/pagopa/afm/marketplacebe/util/CommonUtil.java:32]: se `validityDateTo = now`: `F || !(T || F) => F`
se `validityDateTo = before`: `F || !(F || T) => F`
se `validityDateTo = after`: `F || !(F || F) => T`
se `validityDateTo = null`: `T || !(F || F) => T`

I think it is correct

Comment (jacopocarlini) [reply src/main/java/it/pagopa/afm/marketplacebe/util/CommonUtil.java:32]: Sorry, i missed the brackets

**Review by jacopocarlini — DISMISSED** [review]
good 😄

**Review by jacopocarlini — APPROVED** [review]

**Pull request merged**